docs/guides/kyc.md

```diff
@@ -7,62 +7,177 @@ title: KYC Verification
 
 ## Overview
 
-The Camino Network is bound to revolutionize the way people will travel in the near future. Nevertheless, as the possibilities for new and exciting ways to create and enjoy travel experiences, so too will the methods that ill-intentioned people or organizations might use to take advantage of them. Therefore, Camino uses a KYC service/process to protect travellers and businesses from fraudulence.
+The Camino Network is set to revolutionize travel, and with new and exciting ways to
+create and enjoy travel experiences, come the possibility of ill-intentioned people
+or organizations taking advantage of them. Therefore, Camino uses a KYC service/process
+to protect travellers and businesses from fraud.
 
-## What does KYC mean and how does it work?
+## What is KYC and how does it work?
 
-KYC stands for Know Your Client/Customer, and it is a regulatory process used to ascertain the identity and other credentials of a user. It is becoming legally mandatory in most countries, to ensure that the users are actually who they claim to be and to help against money laundering and the financing of terrorist activities. Moreover, if the users on the Camino network are verified, this will make our endeavors more trustworthy and less risky, thus enhancing the experience.
+KYC stands for Know Your Client/Customer, and it is a regulatory process used to verify
+the identity and other credentials of a user. It is legally mandatory in most countries
+to ensure users are who they claim to be and to help against money laundering and
+the financing of terrorist activities. Additionally, if users on the Camino network are
+verified, it enhances trustworthiness and reduces risk, thereby improving the overall
+experience.
 
-To be verified, the user has to share an id document and pass a liveness check to authenticate. After a successful verification, we save the verified status to the Camino blockchain where it can be picked up by the network.
+To be verified, the user must share an ID document and pass a liveness check to authenticate.
+After successful verification, the verified status is saved to the Camino blockchain where
+it can be picked up by the network.
```

Comment:
Would change to:
After a successful verification, the verified KYC status is associated with the user wallet. It allows other network participants and builders to know which wallets have a verified KYC, without sharing any of the personal KYC information that the user provided.

Reply:
Thanks, updated.